templates/cosmic-microservices.yaml

```diff
@@ -1,4 +1,8 @@
 {{- $context := . -}}
+{{- range .Values.namespaces -}}
```

Comment:
Also, last time I checked, creating a namespace the already existed would fail when installing a chart.